### ISSUE TITLE
Focus management by tab

### ISSUE THREAD

**Issue by frassinier** [elrumordelaluz/reactour#125]
Hello 👋 

I have to deal with some basic a11y issues and I noticed:

First, there is no explicit keyboard trap while showing the tour.
Indeed, if I press <kbd>Tab</kbd> then I focus element of my page and not the tooltip content.
It's pretty hard to navigate by <kbd>Tab</kbd>.

The second point is that buttons have `outline: 0` so you have no visual indication when they are focused or clicked.

Is there a way to deal with that by adding something like [react-focus-lock](https://github.com/theKashey/react-focus-lock)?
If so, I could create a Pull Request.

**Comment by elrumordelaluz** [comment]
Absolutely valid _Issue_ @frassinier, please PR in any case!

**Comment by frassinier** [comment]
👆 here is it!

I can't test `action` step callback with hooks,
And when I would switch to legacy version of Tour, I get white page. ¯\\_(ツ)_/¯

**Comment by elrumordelaluz** [comment]
super! 

> I can't test action step callback with hooks version

Yes, _TourHooks_ is missing some parts, I need to find some time to pass all `props` into it.

> And when I would switch to legacy one, I get white page. ¯\(ツ)/¯

Probably is caused by #121. Please take a look into the _Demo_ that now (not deployed) is usins `lazy` and `Suspense`. Should work in case you change it. Let me know if I understand well.

**Comment by frassinier** [comment]
Yes disabling Lazy and Suspense solves the issue with the white page.
I'm able to test right now, thanks.
And it looks ok for me. :+1:

**Comment by elrumordelaluz** [comment]
should be ready in `reactour@1.13.4`

**Comment by frassinier** [comment]
Thank you Lionel for your quick answers!
